study-builder/studies/osteo/study.conf

```diff
@@ -1631,6 +1686,36 @@
       "dispatchToHousekeeping": false,
       "order": 3
     },
+    {
+      "label": "CREATE_FH_ON_CONSENT_COMPLETE",
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "FAMILY_HISTORY"
+      }
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 4
+    },
+    {
+      "label": "CREATE_FH_ON_CONSENT_ASSENT_COMPLET",
```

Comment:
```suggestion
      "label": "CREATE_FH_ON_CONSENT_ASSENT_COMPLETE",
```

Reply:
Unfortunately the final "E" was left out on purpose. 36 character limit reached. This is first time we use labels. Maybe this is indication that field is too short or the naming convention I used is not good? I am not losing sleep over it, but if you have strong convictions on this?

Reply:
Ah! That explains it! If you're good with this, then let's leave it.